Source/WebCore/page/LocalDOMWindow.cpp

```diff
@@ -2457,6 +2459,46 @@ void LocalDOMWindow::finishedLoading()
     }
 }
 
+LocalDOMWindow::PerformanceEventTimingCandidate LocalDOMWindow::initializeEventTimingEntry(const Event& event, EventTypeInfo typeInfo)
+{
+    LOG_WITH_STREAM(PerformanceTimeline, stream << "Initializing event timing entry of type " << event.type());
+    // FIXME: implement InteractionId logic
+    auto processingStart = performance().now();
+    return PerformanceEventTimingCandidate {
+        .typeInfo = typeInfo,
+        .cancelable = event.cancelable(),
+        // Account for event.timeStamp() unreliability (based on wall clock):
+        .startTime =  std::min(processingStart, performance().relativeTimeFromTimeOriginInReducedResolution(event.timeStamp())),
+        .processingStart = processingStart,
+        .processingEnd = 0,
+        .target = nullptr
```

Comment:
No need to explicitly set processingEnd or target here. They are initialized to 0 & nullptr already.